calico-policies/private-network-isolation/calico-v3/au-syd/allow-egress-pods-private.yaml

```diff
@@ -1,7 +1,8 @@
 # This policy opens port 53 and 5353 for DNS,
-# port 2049 for communication with NFS file servers, ports 443 and 3260
-# for communication to block storage, port 10250 for VPN communication,
-# and port 2040 and 2041 on 172.20.0.0 for the master API server local proxy. It also
+# port 2049 for communication with NFS file servers,
+# ports 443 and 3260 for communication to block storage,
+# port 10250 for VPN pod, port 4443 or 6443 for metrics-server, and
```

Comment:
port 10250 for **the** VPN pod (is this is the openVPN client? of so, maybe call it like that)